projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/utils/first-error.pipe.ts

```diff
@@ -35,7 +35,7 @@ export class FirstErrorPipe implements PipeTransform, OnDestroy {
 
     let errorMessage: string;
     if (keys[0] === 'required') {
-      errorMessage = '%FIELDLABEL% is required';
+      errorMessage = '%FIELDLABEL% Save and continue';
```

Comment:
Why is this change required?

Reply:
This text was used for test purposes only to demonstrate the Welsh translation enhancement works. The phrase "is required" is not currently in the Welsh database. As stated above, the PR is not to be merged.